src/main/java/com/spartronics4915/frc2019/subsystems/PanelHandler.java

```diff
@@ -149,7 +168,18 @@ public synchronized void setWantedState(WantedState wantedState)
 
     public synchronized boolean atTarget()
     {
-        return mSystemState == SystemState.RETRACTING && mWantedState == WantedState.RETRACT;
+        switch (mWantedState)
+        {
+            case ARM_DOWN:
+                return mSystemState == SystemState.ARM_DOWNING;
```

Comment:
Add a `mStateChangedTimer.hasPeriodPassed(Constants.kPanelArmMovementTime)` and associated `private final Timer mStateChangedTimer = new Timer()`.